semantic-go/src/Language/Go.hs

```diff
@@ -1,16 +1,16 @@
 -- | Semantic functionality for Go programs.
 module Language.Go
 ( Term(..)
-, TreeSitter.Go.tree_sitter_go
+, Language.Go.Grammar.tree_sitter_go
 ) where
 
 
 import           Data.Proxy
+import qualified Language.Go.AST as Go
 import qualified Language.Go.Tags as GoTags
 import qualified Tags.Tagging.Precise as Tags
-import qualified TreeSitter.Go (tree_sitter_go)
-import qualified TreeSitter.Go.AST as Go
-import qualified TreeSitter.Unmarshal as TS
+import qualified Language.Go.Grammar (tree_sitter_go)
```

Comment:
I think we could still get this symbol from `TreeSitter.Go` since neither that module nor its export of that symbol are going away.